src/test/java/manandwomantests/WomanTest.java

```diff
@@ -9,19 +9,19 @@ public class WomanTest {
 
     @Test
     public void testIsRetired(){
-        Woman woman = new Woman("Mary", "Smith", 62, "Alex");
-        Assert.assertEquals(woman.isRetired(),55,"Cannot be retired");
+        Woman woman = new Woman("Mary", "Smith", 65, "Alex");
+        Assert.assertTrue(woman.isRetired(),"Can be retired");
 
     }
 
     @Test
     public void testRegisterPartnership(){
             Woman woman = new Woman("Mary", "Smith", 62, "Alex");
-            Assert.assertEquals(woman.registerPartnership(), "Alex", "Registered");
+            ssert.assertEquals(woman.getPartner(), "Register Partnership");
     }
     @Test
     public void testDeregisterPartnership(){
         Woman woman = new Woman("Mary", "Smith", 62, "Alex");
-        Assert.assertEquals(woman.deregisterPartnership(), "Mike", "Deregister");
+        ssert.assertNull(woman.deregisterPartnership());
     }
 }
```

Comment:
@LanaSvit Замінила трохи ассерти, але не впевнена чи правильно. 
P.S. у мене чомусь не відправляються локально зміни сюди на гіт (видає помилку що в терміналі, що в IDEA (error: failed to push some refs to 'https://github.com/annavolovyk/dz-11.git'
hint: Updates were rejected because the tip of your current branch is behind),
тому можливо доведеться створити новий репозитарій, але вже коли виправлю усі коментарі